moveit_tutorials.rosinstall

```diff
@@ -0,0 +1,8 @@
+- git:
+    local-name: moveit_tutorials
+    uri: https://github.com/ros-planning/moveit_tutorials.git
+    version: master
+- git:
```

Comment:
I don't like the approach to build MoveIt as prerequisite of any small dependency package.
We should provide a binary release of melodic and master and use it here.

Reply:
I thought about it and I see some issues with that approach. Maybe you can help me come up with resolutions for them:

1. You are correct that master is failing because it is trying to build against the released melodic debians. If we create new debians then we will have to change the naming in quite a few locations
2. We could create new ci, ci-shadow-fixed and source docker containers for master and run the ci in the source image. (We will want to 
3. Unfortunately, if we build moveit from source in CI then we run up on the free tier time limit. If we change TEST_BLACKLIST to BLACKLIST and use it when building as well as testing that may help.

Reply:
In the future, I prefer the first option, i.e. releasing the master branch as binaries, maybe using our own ppa.
However, meanwhile, option 2 looks best to me. Having a `moveit/moveit:master-source` docker container will save us unnecessary compilation time of MoveIt for dependent packages.
I already added an appropriate docker build tag. We will need to augment moveit_ci to allow a custom docker image.